api/api/versions_api.go

```diff
@@ -115,6 +117,10 @@ func (c *VersionsController) CreateVersion(r *http.Request, vars map[string]stri
 		return BadRequest("Valid label key/values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.")
 	}
 
+	if versionPost.PythonVersion == "" { // if using older sdk, this will be blank
```

Comment:
I think we need to return badrequest if the given python version is not supported.

Reply:
at this stage we have no idea if the model version is supported since we don't have it hardcoded anywhere. This might also not be a batch or pyfunc model, which... may work even if we don't officially support it?

Reply:
As per discussion, the enforcement of python version is delegated to the sdk as per:
https://github.com/gojek/merlin/blob/upgrade-pyfunc/python/sdk/merlin/model.py#L430
https://github.com/gojek/merlin/blob/upgrade-pyfunc/python/sdk/setup.py#L74

For older sdk version, this value is not captured but set to 3.7.* by default as per code above.